code/game/atoms.dm

```diff
@@ -30,6 +30,12 @@
 
 	if(color)
 		add_atom_colour(color, FIXED_COLOUR_PRIORITY)
+
+	if(!atom_colours)
+		atom_colours = list(ADMIN_COLOUR_PRIORITY = null, \
+		TEMPORARY_COLOUR_PRIORITY = null, \
+		WASHABLE_COLOUR_PRIORITY = null, \
+		FIXED_COLOUR_PRIORITY = null)

```

Comment:
The issue is only in the spacevine code because it's the only one that looked for a specific color layer, instead of copying the whole thing, though. If someone adds a feature that needs to check an atom's washable color, for example, they're gonna run into the same issue again, no?
If you want me to, though, i'll fix it like you said.